.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+    - "10"
```

Comment:
Must it really be a string? :innocent:

Reply:
I believe so: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/

Reply:
![](https://media.giphy.com/media/l0MYvffOBO1niIsqA/giphy.gif)

Reply:
![1d374b14141689131240501455_700wa_0](https://user-images.githubusercontent.com/1150201/48866407-0609fb80-edd3-11e8-82ac-31670d73cac1.gif)